frontend/src/components/Profile/ProfilePreview/ProfileMeta/RecruiterProfileMeta.tsx

```diff
@@ -17,6 +19,9 @@ export const RecruiterProfileMeta = React.memo<Props>((props) => {
           className={cn(styles.recruiterMetaItem, typography.smallHeading)}
           key={item.name}
         >
+          {item.name === RecruiterProfileMetaItems.City && (
+            <IconLocation />
+          )}
```

Comment:
what if we want to show icons for other items? try to make the approach more generic and re-usable

Reply:
Thanks for advice, fixed